bin/build_utils

```diff
@@ -25,3 +21,17 @@ function gen_versions() {
     echo $version
   done
 }
+
+# returns the git commit's hash only for non-fixed versions, empty otherwise.
+# which can be identified by the TAG_NAME variable, which is being set automatically on tag-triggered builds
```

Comment:
i would change this sentence to `The TAG_NAME variable is set automatically on tag-triggered builds for non-fixed versions`

Reply:
Actually this is the opposite, when pushing a new tag for a fixed version, a new tag-trigged build starts, which has the TAG_NAME env variable.

Reply:
let me know if you think It's clear thats way

Reply:
yes, please update the comment to say that